packages/world/src/Constants.sol

```diff
@@ -37,9 +37,18 @@ uint128 constant MOVE_ENERGY_COST = 25543750000000;
 uint128 constant PLAYER_FALL_ENERGY_COST = MAX_PLAYER_ENERGY / 25; // This makes it so, with full energy, you die from a 25 block fall
 
 // Resource caps
-uint256 constant MAX_WHEAT_SEED = 5_000_000;
-uint256 constant MAX_OAK_SEED = 1_000_000;
-uint256 constant MAX_SPRUCE_SEED = 1_000_000;
+uint256 constant MAX_WHEAT_SEED = 444_971;
```

Comment:
We should update the new energy buffer value in this PR as well

Reply:
could you remind me how we calculated that before?

Reply:
should just be `Energy Buffer in each Energy Region` cell times the factor

Reply:
also looks like `max player energy bar` changed

Reply:
actually a bunch changed, could do in follow up PR lol nvm